ChangeLog

```diff
@@ -1,3 +1,9 @@
+Changes in squid-6.0.2 (30 Apr 2023):
+
+	- Avoid excessive disk I/O in some environments
+	- ... and several build and portability fixes
+	- ... and all fixes from 5.9
```

Comment:
No, v6.0.2 does not actually contain "all fixes from 5.9" -- some (possibly "half" or even "most", depending on how one counts changes) v5.9 changes are specific to the v5 branch. As you know, sending readers on a chase across change log blobs is bad UX IMO. In this particular case, the reference is also factually wrong AFAICT.